yarn.lock

```diff
@@ -10340,7 +10340,7 @@ cacheable-request@^6.0.0:
   dependencies:
     clone-response "^1.0.2"
     get-stream "^5.1.0"
-    http-cache-semantics "^4.0.0"
+    http-cache-semantics "^4.1.1"
```

Comment:
I suspect CI won't like this change, but let's see. Leaving the existing clause in here should be sufficient for us to pull in `4.1.1`

Reply:
hmm, but CI accepted the change 🤔